### PR TITLE
rename methods

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

**Comment by nicolasThal** [comment]
https://github.com/open-orchestra/open-orchestra-mongo-libs/pull/20
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/529
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1452
https://github.com/open-orchestra/open-orchestra-model-interface/pull/166
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/165
https://github.com/open-orchestra/open-orchestra-display-bundle/pull/199
https://github.com/open-orchestra/open-orchestra-front-bundle/pull/143

**Comment by veolys-nanne** [comment]
:+1:

File: FrontBundle/Tests/Routing/DatabaseRouteLoaderTest.php [hunk 1]
```diff
@@ -112,8 +112,8 @@ public function testLoad()
         $enNodes[] = $enGrandSonNode;
 
         // Define the repository return
-        Phake::when($this->nodeRepository)->findLastPublishedVersion($this->frLocale, $siteId)->thenReturn($frNodes);
-        Phake::when($this->nodeRepository)->findLastPublishedVersion($this->enLocale, $siteId)->thenReturn($enNodes);
+        Phake::when($this->nodeRepository)->findCurrentlyPublishedVersion($this->frLocale, $siteId)->thenReturn($frNodes);
```
Comment (ngilain) [inline FrontBundle/Tests/Routing/DatabaseRouteLoaderTest.php:115]: findCurrentlyPublishedVersion or findOneCurrentlyPublishedVersion ?

File: FrontBundle/Tests/Routing/DatabaseRouteLoaderTest.php [hunk 2]
```diff
@@ -112,8 +112,8 @@ public function testLoad()
...
+        Phake::when($this->nodeRepository)->findCurrentlyPublishedVersion($this->enLocale, $siteId)->thenReturn($enNodes);
```
Comment (ngilain) [inline FrontBundle/Tests/Routing/DatabaseRouteLoaderTest.php:116]: same question

File: FrontBundle/Tests/Routing/DatabaseRouteLoaderTest.php [hunk 3]
```diff
@@ -176,7 +176,7 @@ public function testLoadWithFullUrl()
         $frNodes[] = $frGrandSonNode;
 
         // Define the repository return
-        Phake::when($this->nodeRepository)->findLastPublishedVersion($this->frLocale, $siteId)->thenReturn($frNodes);
+        Phake::when($this->nodeRepository)->findCurrentlyPublishedVersion($this->frLocale, $siteId)->thenReturn($frNodes);
```
Comment (ngilain) [inline FrontBundle/Tests/Routing/DatabaseRouteLoaderTest.php:179]: same question

File: FrontBundle/Tests/Routing/DatabaseRouteLoaderTest.php (hunk 1 above)
Comment (nicolasThal) [inline FrontBundle/Tests/Routing/DatabaseRouteLoaderTest.php:115]: I return an array of elements, so I would let findCurrentlyPublishedVersion

**Comment by nicolasThal** [comment]
@itkg-nanne pr updated

**Comment by veolys-nanne** [comment]
:+1:

**Pull request merged**